src/services/ClientService.ts

```diff
@@ -25,22 +25,51 @@ import { ServiceLocator } from './ServiceLocator';
 import { CLIService } from './CLIService';
 import { Errors } from '../Errors/LocalNodeErrors';
 
+/**
+ * Represents a service for managing the Hedera client.
+ * @implements {IService}
+ */
 export class ClientService implements IService{
+    /**
```

Comment:
Should we leave those in every ts class, since some of them are repetitive and self-explanatory. E.g logger service, serviceName
Just asking, not sure what is good practice

Reply:
The best practice is to move them to the a base class. But that is not the focus here.
Since we are discussing most of the comments are. I think there is no reason to have comments in TypeScript at all.  :)

Reply:
@georgi-l95 what do you think?

Reply:
Let's leave it for consistency.